src/components/table/index.tsx

```diff
@@ -201,78 +201,76 @@ export default function EnhancedTable<T>({
   return (
     <>
       {showTopBar && <EnhancedTableToolbar numSelected={selectedRows ? selectedRows.length : 0} topBarButtons={topBarButtons} />}
-      <TableContainer id={id}>
```

Comment:
Changes here are only the removal of `TableContainer` and unindenting its children